<ui version="4.0" >
 <class>CollectionEditor</class>
 <widget class="QDialog" name="CollectionEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Collection editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="m_nameLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Collection name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLineEdit" name="m_nameEdit" >
     <property name="toolTip" >
      <string>Name of the function being edited</string>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="1" column="0" colspan="2" >
    <widget class="QTreeWidget" name="m_tree" >
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Function</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QToolButton" name="m_add" >
     <property name="toolTip" >
      <string>Add function(s) to the collection</string>
     </property>
     <property name="text" >
      <string notr="true">...</string>
     </property>
     <property name="icon" >
      <iconset resource="main.qrc" >
       <normaloff>:/edit_add.png</normaloff>:/edit_add.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QToolButton" name="m_remove" >
     <property name="toolTip" >
      <string>Remove the selected function</string>
     </property>
     <property name="text" >
      <string notr="true">...</string>
     </property>
     <property name="icon" >
      <iconset resource="main.qrc" >
       <normaloff>:/edit_remove.png</normaloff>:/edit_remove.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="m_buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="main.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CollectionEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CollectionEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
